playbooks/set-gpu-clocks.yml

```diff
@@ -0,0 +1,16 @@
+---
+# set gpu clocks on all worker nodes
+- hosts: compute-node
+  gather_facts: no
+  become: yes
+  vars:
```

Comment:
These variables should go in `group_vars` somewhere instead of the playbook. If you want to set defaults, we should create a role.

Reply:
Some kind of "gpu management" role might be useful where we could set clocks, reset GPUs, etc.

Reply:
I agree on the gpu management role and this being rolled in eventually. This was just a quick and dirty playbook people have used for some benchmarking use cases.

We can make this a role, but there's really only 1 variable people will ever change. Max clock or clock reset.

Reply:
There are tons of other tunable GPU parameters we could set via a role, I think it would be really useful.